### PR TITLE
csv for PatientRegistration

### PR DESCRIPTION
**Pull request by rejuls**

### CODE REVIEW

**Comment by rejuls** [comment]
@vigneshhari

File: care/facility/api/viewsets/patient.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 import json
 from json import JSONDecodeError
 from config.celery_app import app
-
+from django.conf import settings
 from django.contrib.postgres.search import TrigramSimilarity
 from django.core.validators import validate_email
 from django.db.models.query_utils import Q
 from django_filters import rest_framework as filters
+from djqscsv import render_to_csv_response
```
Comment (vigneshhari) [inline care/facility/api/viewsets/patient.py:10]: Codacy found an issue: [third party import "from djqscsv import render_to_csv_response" should be placed before "from config.celery_app import app"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)

Comment (vigneshhari) [inline care/facility/api/viewsets/patient.py:10]: Codacy found an issue: [Unable to import 'djqscsv'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)

File: care/facility/api/viewsets/patient.py [hunk 2]
```diff
@@ -2,11 +2,12 @@
 import json
 from json import JSONDecodeError
 from config.celery_app import app
-
+from django.conf import settings
```
Comment (vigneshhari) [inline care/facility/api/viewsets/patient.py:5]: Codacy found an issue: [third party import "from django.conf import settings" should be placed before "from config.celery_app import app"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)

File: care/facility/models/patient.py [hunk 3]
```diff
@@ -192,6 +193,36 @@ def save(self, *args, **kwargs) -> None:
...
+        "past_travel" : "Travelled to Any Foreign Countries in the last 28 Days",
+        "countries_travelled" : "Countries Patient has Travelled to",
+        "date_of_return" : "Return Date from the Last Country if Travelled",
+        "present_health" : "Patient's Current Health Details",
+        "ongoing_medication" : "Already pescribed medication if any",
+        "has_SARI" : "Does the Patient Suffer from SARI",
+        "date_of_receipt_of_information" : "Patient's information received date",
+    }
+
+    CSV_MAKE_PRETTY = {"gender": (lambda x: REVERSE_GENDER_CHOICES[x])}
```
Comment (vigneshhari) [inline care/facility/models/patient.py:225]: Codacy found an issue: [Undefined variable 'REVERSE_GENDER_CHOICES'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5753282)

File: care/facility/api/viewsets/patient.py [hunk 4]
```diff
@@ -128,6 +129,11 @@ def list(self, request, *args, **kwargs):
             EXPIRED = 6
 
         """
+        if settings.CSV_REQUEST_PARAMETER in request.GET:
+            queryset = self.filter_queryset(self.get_queryset()).values(*PatientRegistration.CSV_MAPPING.keys())
+            return render_to_csv_response(
+                queryset, field_header_map=PatientRegistration.CSV_MAPPING
+            )
```
Comment (nidhinkandoth) [inline care/facility/api/viewsets/patient.py:136]: Specify `field_serializer_map`

File: care/facility/models/patient.py (hunk 3 above)
Comment (nidhinkandoth) [inline care/facility/models/patient.py:225]: Suggest to add `date_of_birth`, `estimated_contact_date`, `date_of_receipt_of_information`, `date_of_return`,
These datetime fields in CSV_MAKE_PRETTY to make it more readable. 
Ref https://pypi.org/project/django-queryset-csv/

**Pull request merged**